cves/CVE-2013-4315.yml

```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In versions 1.4.x and 1.5.x of Django, an oversight in ssi template tags granted
+  user inputed files unauthroized access to root directories. When reading in arbitrary
+  files it allowed directory traversal, potentially exposing directories containing 
+  sensitive information such as passwords.
```

Comment:
How can you securely use a ssi tag?

Reply:
The ssi tag is securely used by inputting an absolute path.